cli/compose/types/types.go

```diff
@@ -296,6 +296,11 @@ type ServiceVolumeVolume struct {
 	NoCopy bool `mapstructure:"nocopy"`
 }
 
+// ServiceVolumeTmpfs are options for a service volume of type tmpfs
+type ServiceVolumeTmpfs struct {
+	Size int64
```

Comment:
This should probably be an `uint64`? Or is a signed integer used elsewhere?

Reply:
Chose to use int64 over unit64 because the mount struct looked to expect TmpfsOptions to contain it. Source:
https://github.com/moby/moby/blob/6415f1dcf5c79da853c604733c56b43ae3d18ced/api/types/mount/mount.go#L108

Reply:
Thanks! Wonder if that was for a reason ("special `-1` value, e.g.) or just overlooked during review of the initial implementation. 🤔